R/transformCounts.R

```diff
@@ -40,10 +40,26 @@
 #' \item{'clr'}{ Centered log ratio (clr) transformation can be used for reducing the
 #' skewness of data and for centering it. (See e.g. Gloor et al. 2017.)
 #'
-#' \deqn{clr = log_{10}x_{r} - log_{10}µ_{r}}{%
-#' clr = log10 x_r - log10 µ_r}
-#' where \eqn{x_{r}} is a single relative value, \eqn{\mu_{r}} is
-#' mean relative value".}
+#' \deqn{clr = log_{10}\frac{x_{r}}{g(x_{r})} = log_{10}x_{r} - log_{10}µ_{r}}{%
+#' clr = log10(x_r/g(x_r)) = log10 x_r - log10 µ_r}
+#' where \eqn{x_{r}} is a single relative value, g(x_{r}) is geometric mean of
+#' sample-wide relative values, and \eqn{\mu_{r}} is arithmetic mean of 
+#' sample-wide relative values".}
+#' 
+#' \item{'clr_robust'}{ Robust clr is similar to regular clr. Problem of regular
+#' clr is that logarithmic transformations leads to undefined values when zeros
+#' are present in the data. In robust clr, values are divided by geometric mean
+#' of observed taxa and zero values are not taken into account. Zero values will
+#' stay as zeroes. 
+#' 
+#' Because of high-dimensionality of data, robust clr's geometric mean of 
```

Comment:
Refer to the method as "rclr" instead of "robust clr" so that it is clear it is uniquely defined (in principle there could be many "robust" ways to implement clr)

Reply:
ok